environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -105,6 +104,50 @@ func collectStats(container Container) {
 	}()
 }
 
+func collectNetworkStats(container Container, client docker.Client) {
+	go func() {
+		foNet, err := os.Create("/app/"+container.Name+"_network_tmp")
+		foTop, err := os.Create("/app/"+container.Name+"_top_tmp")
+		if err != nil {
+	        panic(err)
+	    }
+		interfaces, err := net.Interfaces()
```

Comment:
nethogs already runs in its own routine, because of the way the golang cmd exec works. When it starts, it runs on its own and writes into the file.